ntex-codec/Cargo.toml

```diff
@@ -17,11 +17,12 @@ path = "src/lib.rs"
 
 [dependencies]
 bitflags = "1.3"
-ntex-bytes = "0.1"
-ntex-util = "0.1"
+futures-util = { version = "0.3", default-features = false }
+ntex-bytes = { version = "0.1", path = "../ntex-bytes/" }
```

Comment:
this changes are not required, top level Cargo.toml contains patch section

Reply:
You're right. I saw it compiling crates.io ntex and thought it was because of the missing `path`. In fact it was just because of the dev-dep pointing to the old version. I've reverted it.